<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileItem</class>
 <widget class="QWidget" name="FileItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>206</width>
    <height>81</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>81</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>206</width>
    <height>84</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>206</width>
     <height>75</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,5">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="avatar_lab">
      <property name="minimumSize">
       <size>
        <width>36</width>
        <height>36</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>36</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>头像</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="fileName">
        <property name="styleSheet">
         <string notr="true">font: 25 9pt &quot;Microsoft YaHei&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
	background-color:transparent;
}

QPushButton{color: rgb(85, 170, 255);}

QPushButton:hover{color: rgb(85, 170, 100);}
QPushButton:pressed{color: rgb(85, 170, 0);}

</string>
          </property>
          <property name="text">
           <string>另存为</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sendOrRecv_btn">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
	background-color:transparent;
}

QPushButton{color: rgb(85, 170, 255);}

QPushButton:hover{color: rgb(85, 170, 100);}
QPushButton:pressed{color: rgb(85, 170, 0);}

</string>
          </property>
          <property name="text">
           <string>发送</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancel_btn">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
	background-color:transparent;
}

QPushButton{color: rgb(85, 170, 255);}

QPushButton:hover{color: rgb(85, 170, 100);}
QPushButton:pressed{color: rgb(85, 170, 0);}

</string>
          </property>
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
